Client/Frontend/Widgets/HighlightCell.swift

```diff
@@ -197,3 +197,70 @@ class HighlightCell: UITableViewCell {
         self.timeStamp.text = "3 days ago"
     }
 }
+
+struct HighlightIntroCellUX {
+    static let foxImageName = "fox_finder"
+    static let margin: CGFloat = 20
+}
+
+class HighlightIntroCell: UITableViewCell {
+
+    lazy var titleLabel: UILabel = {
+        let textLabel = UILabel()
+        textLabel.font = DynamicFontHelper.defaultHelper.DeviceFontMediumBold
+        textLabel.textColor = UIColor.blackColor()
+        textLabel.numberOfLines = 1
+        textLabel.adjustsFontSizeToFitWidth = true
+        textLabel.minimumScaleFactor = 0.8
+        return textLabel
+    }()
+
+    lazy var mainImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: HighlightIntroCellUX.foxImageName)
+        return imageView
+    }()
+
+    lazy var descriptionLabel: UILabel = {
+        let label = UILabel()
+        label.font = DynamicFontHelper.defaultHelper.DeviceFontDescriptionActivityStream
+        label.textColor = UIColor.darkGrayColor()
+        label.numberOfLines = 0
+        return label
+    }()
+
+    override init(style: UITableViewCellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+        self.selectionStyle = .None
+        contentView.addSubview(titleLabel)
+        contentView.addSubview(mainImageView)
+        contentView.addSubview(descriptionLabel)
+
+        titleLabel.text = Strings.HighlightIntroTitle
+        descriptionLabel.text = Strings.HighlightIntroDescription
+
+        titleLabel.snp_makeConstraints { make in
+            let insets = UIEdgeInsets(top: HighlightIntroCellUX.margin, left: HighlightIntroCellUX.margin, bottom: 0, right: 0)
```

Comment:
you can bring this outside the makeConstraints closure and label titleLabelInsets, idk it just looks cleaner to leave it out of there - or you can just put the Insets object directly